getting_started/first_2d_game/07.finishing-up.rst

```diff
@@ -33,8 +33,15 @@ Add two :ref:`AudioStreamPlayer <class_AudioStreamPlayer>` nodes as children of
 click on the ``Stream`` property, select "Load", and choose the corresponding
 audio file.
 
-To play the music, add ``$Music.play()`` in the ``new_game()`` function and
-``$Music.stop()`` in the ``game_over()`` function.
+All audio is automatically imported with the "Loop" setting disabled.
+The "art/House In a Forest Loop.ogg" music file should have "Loop" checked to
+**On** and the "art/gameover.wav" should have the "Loop" not checked. Next to the
+"Scene" tab (left side of the editor), click on the "Import" tab. In the "FileSystem"
+window below that click on the music file under "art" and on the "Import" tab check
+the box next to "Loop", so that the music file will now loop automatically.
+
+To play the music, add ``$Music.play()`` in the ``_on_start_timer_timeout()``
```

Comment:
```suggestion
To play the music, add ``$Music.play()`` in the ``new_game()``
```

Reverting this small change from the original as I don't fully get the reasoning and have no time to test the tutorial code with this change, and the alternative PR (which does less than this one) https://github.com/godotengine/godot-docs/pull/7228/files did not make this change and it seems the previous code should still work here.